api/src/test/resources/org/openmrs/api/include/UserServiceTest.xml

```diff
@@ -63,4 +63,10 @@
+
+	<person person_id="5512" gender="M" dead="false" creator="1" date_created="2021-09-02 15:46:47.0" voided="false" uuid="d6bb960d-40c1-49d8-b58c-aa19adb4cb63"/>
+	<users user_id="5512" person_id="5512" system_id="512-x" username="test1" password="e9fb6ffdc1f903bdd8ae621347d5329762de868347d317dacecebf6e9e73f4d97297cb1237d8edaa02c5a9bd25949b251c2169202236455c23cdf4d828487c4a" salt="b3cc5eea8fdc9f8579b07267f209fe184f2202cc6379922e0b13f2753c641808bbba8b0ecf4d0592db8e5001e57876c506ae4d3579d5274d88ca8f26b7e14d9d" secret_question="a secret" secret_answer="an answer" creator="1" date_created="2021-09-02 15:46:47.0" changed_by="1" date_changed="2021-09-04 15:46:47.0" retired="false" retire_reason="" uuid="eee36210-0a6b-4325-965a-e73d423df2e3"/>
+
+	<user_property user_id="5512" property="emrapi.lastViewedPatientIds" property_value="479, 93, 442, 434, 341, 295, 384, 2, 97, 363, 267, 458, 369, 221, 28, 359, 53, 196, 416, 31, 422, 15, 173, 412, 311, 122, 194, 343, 3, 133, 498, 362, 134, 26, 158, 181, 43, 309, 172, 98, 379, 382, 298, 136, 451, 334, 81, 99, 388, 378, 138, 42, 331, 326, 393, 65, 355, 68, 111, 147, 300, 413, 20, 57, 426, 84, 206, 294, 83, 404, 390, 389, 289, 272, 166, 88, 180, 283, 472, 276, 401, 425, 12, 127, 49, 459, 153, 261, 186, 102, 320, 63, 164, 346, 222, 160, 375, 465, 485, 301, 457, 497, 168, 470, 46, 243, 8, 137, 100, 108, 489, 336, 32, 312, 323, 239, 90, 25, 198, 350, 234, 432, 371, 468, 215, 228, 352, 140, 170, 128, 268, 481, 157, 257, 92, 86, 492, 176, 408, 135, 347, 10, 278, 344, 203, 4, 24, 69, 269, 333, 462, 260, 431, 74, 491, 325, 396, 197, 244, 455, 281, 423, 48, 313, 330, 175, 406, 188, 148, 424, 445, 297, 124, 339, 418, 377, 179, 79, 144, 29, 50, 328, 411, 163, 209, 421, 493, 251, 453, 232, 480, 219, 115, 376, 322, 116, 200, 235, 469, 262, 345, 358, 366, 123, 36, 154, 360, 114, 249, 460, 22, 62, 113, 254, 471, 201, 230, 177, 129, 280, 314, 467, 66, 383, 76, 77, 155, 351, 21, 449, 187, 192, 214, 446, 241, 183, 162, 307, 429, 73, 452, 30, 486, 490, 259, 161, 52, 231, 308, 420, 357, 174, 210, 47, 211, 263, 103, 44, 55, 305, 204, 318, 39, 405, 120, 247, 286, 398, 466, 54, 342, 126, 273, 477, 265, 430, 236, 217, 149, 252, 302, 178, 270, 340, 433, 417, 332, 101, 60, 216, 242, 195, 11, 303, 387, 415, 365, 110, 224, 319, 184, 324, 367, 226, 271, 33, 474, 38, 64, 282, 233, 264, 428, 372, 14, 258, 185, 438, 171, 72, 18, 478, 463, 152, 212, 87, 118, 27, 494, 145, 91, 293, 356, 327, 169, 109, 285, 125, 381, 354, 237, 386, 397, 16, 17, 450, 380, 207, 437, 487, 368, 394, 202, 58, 454, 291, 80, 104, 296, 85, 475, 218, 40, 167, 310, 364, 156, 410, 337, 248, 288, 146, 5, 444, 78, 225, 329, 82, 407, 253, 414, 35, 205, 165, 121, 151, 419, 89, 141, 106, 190, 321, 290, 409, 67, 159, 482, 59, 266, 403, 117, 182, 95, 199, 500, 71, 335, 142, 400, 255, 275, 6, 9, 315, 94, 370, 484, 107, 488, 399, 391, 75, 246, 213, 189, 306, 392, 483, 240, 7, 499, 287, 130, 112, 473, 56, 279, 385, 402, 19, 70, 274, 51, 395, 143, 448, 439, 299, 374, 150" />
```

Comment:
Why such a very long property value?

Reply:
I know it is ugly but i used it to prove that the new property_value can actually accommodate a longtext string .Is it bad practice ? if so should i truncate ?

Reply:
Can you load the object and then assign the property value in code?

Reply:
No what i actually did was compare the property_value in the code with the one which I provided for in the dataset.